workspace/apps/pidp/src/assets/snowplow.js

```diff
@@ -0,0 +1,38 @@
+// <!-- Snowplow starts plowing - Standalone vE.2.14.0 -->
+(function (p, l, o, w, i, n, g) {
+  if (!p[i]) {
+    p.GlobalSnowplowNamespace = p.GlobalSnowplowNamespace || [];
+    p.GlobalSnowplowNamespace.push(i);
+    p[i] = function () {
+      (p[i].q = p[i].q || []).push(arguments);
+    };
+    p[i].q = p[i].q || [];
+    n = l.createElement(o);
+    g = l.getElementsByTagName(o)[0];
+    n.async = 1;
+    n.src = w;
+    g.parentNode.insertBefore(n, g);
+  }
+})(
+  window,
+  document,
+  'script',
+  'https://www2.gov.bc.ca/StaticWebResources/static/sp/sp-2-14-0.js',
+  'snowplow',
+);
+var collector = 'spm.apps.gov.bc.ca';
+window.snowplow('newTracker', 'rt', collector, {
+  appId: 'Snowplow_standalone',
+  cookieLifetime: 86400 * 548,
+  platform: 'web',
```

Comment:
Just curious. Then it will report only in web not in mobile ?

Reply:
In this case Snowplow is referring to 'web' as client side tracking. [Reference docs](https://docs.snowplow.io/docs/collecting-data/collecting-from-own-applications/javascript-trackers/)